### PR TITLE
Added support for hl. fixes #131

### LINKED ISSUE #131: munit not working on HashLink target
Can you please have a look at https://github.com/HaxeFoundation/hashlink/issues/52

I cannot determine whether this is a problem in HL or munit.

Thanks.

### PR DESCRIPTION
**Pull request by SlavaRa**
Depends on ~#143~ #145

### CODE REVIEW

**Comment by SlavaRa** [comment]
fixes #131

File: src/massive/munit/client/HTTPClient.hx [hunk 1]
```diff
@@ -285,11 +287,12 @@ class URLRequest
 		#if(js || neko || cpp || java || cs || python || php || hl)
 		client.onData = onData;
 		client.onError = onError;
-			#if(js && !nodejs)
+			#if(js && !hxnodejs)
```
Comment (elsassph) [inline src/massive/munit/client/HTTPClient.hx]: That's a bit odd to see changes which have been merged already - normal?

File: src/massive/munit/client/HTTPClient.hx [hunk 2]
```diff
@@ -309,3 +312,153 @@ class URLRequest
 	}
 	#end
 }
+
+#if hl
+class Http extends haxe.Http {
+	override function readHttpResponse(api:haxe.io.Output, sock:sys.net.Socket) {
+		// READ the HTTP header (until \r\n\r\n)
```
Comment (elsassph) [inline src/massive/munit/client/HTTPClient.hx]: Is there a bug in `haxe.Http` for HL?

Comment (SlavaRa) [reply src/massive/munit/client/HTTPClient.hx]: yes(

**Comment by SlavaRa** [comment]
Done

**Comment by SlavaRa** [comment]
@elsassph all ok?

**Comment by elsassph** [comment]
Approved - would just like a comment from @ncannasse on the Http class

**Comment by ncannasse** [comment]
There is known bug in Http class for HL, as far as I know.

**Comment by SlavaRa** [comment]
https://github.com/HaxeFoundation/haxe/issues/6777

**Pull request merged**